lib/strategies/accept_header_strategy.rb

```diff
@@ -0,0 +1,42 @@
+module Phenotype
+  class AcceptHeaderStrategy
+    attr_reader :env, :content_type, :version_key
+
+    def initialize(app_name: 'rentpath',
+                   version_key: 'version', 
+                   format: 'json')
+      @version_key = version_key
+      @content_type = build_content_type(app_name, format)
+    end
+
+    def get_version(env)
```

Comment:
maybe i am missing something. do we need another header strategy for this? 

why not pass this in from API using header strategy? since project is public, I like the flexibility of changing the header using the existing strategy

```ruby
      api = Phenotype::VersionedApp.new({
        strategies: [Phenotype::HeaderStrategy.new({header: API_HEADER})]
      })
     api.add_version(version: "application/vnd.rentpath.api+json;version=#{my_api_version}", default: false) do |env|
       # etc....
     end
```

this way we can define version schema via api